README.md

```diff
@@ -391,6 +392,32 @@ For more examples and ideas, visit:
 
 ```
 
+#### Proxy Setup
+
+If the device is located behind a corporate proxy you may need to set the correct environment variables.
```

Comment:
In the Docker documentation there are two ways to set a proxy,

- configuring the daemon
- setting environment variables

These instructions configures the daemon, so it seems a bit strange to mention environment variables.

Maybe something like this?
```suggestion
If the device is located behind a proxy the Docker daemon needs to be configured.
```